### PR TITLE
Translate docs/gatsby-cli

### PR DESCRIPTION
**Pull request by bangun-jds**
Finished translating `docs/gatsby-cli`. Please kindly review. Thank you!

### CODE REVIEW

File: docs/docs/gatsby-cli.md [hunk 1]
```diff
@@ -1,19 +1,19 @@
 ---
-title: Commands (Gatsby CLI)
+title: Perintah (Gatsby CLI)
```
Comment (coroo) [inline docs/docs/gatsby-cli.md:2]: Change title with “judul”

**Review by coroo — CHANGES_REQUESTED** [review]
Good job bro!
Just one minor changes please 😁
Thank you

File: docs/docs/gatsby-cli.md (hunk 1 above)
Comment (bangun-jds) [reply docs/docs/gatsby-cli.md:2]: wouldn't that affect when querying the frontmatter?

Comment (coroo) [reply docs/docs/gatsby-cli.md:2]: Actually there is no rule for not changing that part.
But for now, let we keep it.

**Review by coroo — APPROVED** [review]

**Pull request merged**